### PR TITLE
Add icons to the settings tab for better readability

### PR DESCRIPTION
**Pull request by RedsTom**
image

Added icons to every settings element to improve information processing when reading the screen

### CODE REVIEW

**Comment by DennisBauer** [comment]
Thanks for your PR! I like the idea of adding icons to the settings.

File: app/src/commonMain/kotlin/ui/SettingsScreen.kt [hunk 1]
```diff
@@ -79,21 +86,22 @@ fun SettingsScreen(
...
                     title = stringResource(Res.string.settings_default_currency),
                     subtitle =
-                        viewModel.selectedCurrencyName.ifEmpty {
-                            stringResource(Res.string.settings_system_default)
-                        },
+                    viewModel.selectedCurrencyName.ifEmpty {
+                        stringResource(Res.string.settings_system_default)
+                    },
                     onClick = viewModel::onSelectCurrency,
+                    icon = Icons.Rounded.CurrencyExchange,
```
Comment (DennisBauer) [inline app/src/commonMain/kotlin/ui/SettingsScreen.kt:104]: Is there a reason you used the currency exchange icon instead of just the dollar sign here?

**Comment by DennisBauer** [comment]
You have quite some formatting together with your actual code changes. Please fix them by running the Gradle task `ktlintFormat` which should fix all of them automatically.

**Review by DennisBauer — CHANGES_REQUESTED** [review]

File: app/src/commonMain/kotlin/ui/SettingsScreen.kt (hunk 1 above)
Comment (RedsTom) [reply app/src/commonMain/kotlin/ui/SettingsScreen.kt:104]: Not really, can change that if you want

**Comment by RedsTom** [comment]
Screenshot_20240719_143606

Here is the new look of the Default Currency entry

File: app/src/commonMain/kotlin/ui/SettingsScreen.kt (hunk 1 above)
Comment (DennisBauer) [reply app/src/commonMain/kotlin/ui/SettingsScreen.kt]: Using `Paid` also doesn't look that great. So maybe the one you had was better fitting.

**Review by DennisBauer — APPROVED** [review]

**Pull request merged**